apps/juicebox_web/assets/js/search/search.scss

```diff
@@ -2,3 +2,14 @@
   width: 100%;
   margin: 5px;
 }
+
+.result {
```

Comment:
You could just do a separate `results.scss` to prevent having to pass it down 3 layers, but if you think you may want it to look different in different places then I think this is best.